deployment/terraform/variables.tf

```diff
@@ -15,9 +15,9 @@ variable "SUBSCRIPTION_ID" {
 }
 
 variable "K8S_KUBE_CONFIG" {
-  description = "Path to Kube Config File"
+  description = "Path to Kube Config File (can be created through prepare environment)"
 }
 
 variable "K8S_HELM_HOME" {
-  description = "Path to Helm Home Directory"
+  description = "Path to Helm Home Directory (e.g. under $HOME/.helm) and can be created through prepare environment.sh"
```

Comment:
Thanks for adding this hint! "prepare environment.sh" could be changed to "prepare_environment.sh" to make most clear what the name of the script is.